examples/exex/discv5/src/main.rs

```diff
@@ -0,0 +1,29 @@
+use clap::Parser;
+
+use exex::ExEx;
+use network::{cli_ext::Discv5ArgsExt, DiscV5ExEx};
+use reth_node_ethereum::EthereumNode;
+
+mod exex;
+mod network;
+
+fn main() -> eyre::Result<()> {
+    reth::cli::Cli::<Discv5ArgsExt>::parse().run(|builder, args| async move {
+        let tcp_port = args.tcp_port;
+        let udp_port = args.udp_port;
+
+        let handle = builder
+            .node(EthereumNode::default())
+            .install_exex("exex-discv5", move |ctx| async move {
+                // start Discv5 task
+                DiscV5ExEx::new(tcp_port, udp_port).await?;
```

Comment:
it gets dropped immediately, no? I think it will stop the task

Reply:
nice catch, yes we need to remove the `;`

Reply:
I think we should rather save the instance of `DiscV5ExEx` inside the ExEx struct as a field, so it gets dropped at the same time as the ExEx.